NuGet.Config

```diff
@@ -54,12 +59,102 @@
+      <package pattern="Microsoft.CodeAnalysis.Features" />
+      <package pattern="Microsoft.CodeAnalysis.PublicApiAnalyzers" />
+      <package pattern="Microsoft.CodeAnalysis.Scripting.Common" />
+      <package pattern="Microsoft.CodeAnalysis.VisualBasic.Workspaces" />
+      <package pattern="Microsoft.CodeAnalysis.Workspaces.Common" />
+      <package pattern="Microsoft.CSharp" />
+      <package pattern="Microsoft.DataAI.NuGetRecommender.Contracts" />
+      <package pattern="Microsoft.DiaSymReader" />
+      <package pattern="Microsoft.DotNet.Build.Tasks.Feed" />
+      <package pattern="Microsoft.DotNet.Maestro.Tasks" />
+      <package pattern="Microsoft.Extensions.FileProviders.Abstractions" />
+      <package pattern="Microsoft.Extensions.FileSystemGlobbing" />
+      <package pattern="Microsoft.Extensions.Primitives" />
       <package pattern="microsoft.internal.visualstudio.*" />
+      <package pattern="Microsoft.Internal.VisualStudio.Shell.Framework" />
+      <package pattern="Microsoft.IO.Redist" />
+      <package pattern="Microsoft.Net.Compilers.netcore" />
+      <package pattern="Microsoft.Net.Compilers.Toolset" />
+      <package pattern="Microsoft.NET.StringTools" />
+      <package pattern="Microsoft.NET.Test.Sdk" />
+      <package pattern="Microsoft.NETCore.Platforms" />
+      <package pattern="Microsoft.PowerShell.3.ReferenceAssemblies" />
       <package pattern="microsoft.servicehub.*" />
+      <package pattern="Microsoft.ServiceHub.Framework" />
+      <package pattern="Microsoft.TeamFoundationServer.ExtendedClient" />
       <package pattern="microsoft.test.apex.visualstudio" />
+      <package pattern="Microsoft.TestPlatform.Portable" />
+      <package pattern="Microsoft.TestPlatform.TestHost" />
       <package pattern="microsoft.visualstudio.*" />
+      <package pattern="Microsoft.VisualStudio.ComponentModelHost" />
+      <package pattern="Microsoft.VisualStudio.LanguageServices" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Interop" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed.VS" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.VS" />
+      <package pattern="Microsoft.VisualStudio.SDK" />
+      <package pattern="Microsoft.VisualStudio.Sdk.TestFramework.Xunit" />
+      <package pattern="Microsoft.VisualStudio.Setup.Configuration.Interop" />
+      <package pattern="Microsoft.VisualStudio.Shell.15.0" />
+      <package pattern="Microsoft.VisualStudio.VCProjectEngine" />
+      <package pattern="Microsoft.VisualStudio.Workspace.VSIntegration" />
+      <package pattern="Microsoft.VisualStudioEng.MicroBuild.Core" />
+      <package pattern="Microsoft.VSSDK.BuildTools" />
+      <package pattern="Microsoft.Web.Xdt" />
+      <package pattern="Microsoft.Win32.Registry" />
```

Comment:
I think I missed these two from all my other comments, but if the VS SDK have a transitive dependency on either of these packages (registry seems feasible, web.xdt feels very unlikely), then I'd expect them to depend on versions on nuget.org, not VS specific versions.